contrib/containerized-pbench/setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [[ -z "${1}" ]]; then
+    echo "Please provide a target Pbench Server (<host name>:<port number>) to move the results to as the first argument." >&2
+    exit 1
```

Comment:
I would vote for `2` instead of `1` (but `1` is obviously fine).

Reply:
Fixed.